src/main/webapp/app/exam/manage/exams/exam-exercise-import/exam-exercise-import.component.ts

```diff
@@ -33,8 +33,8 @@ export class ExamExerciseImportComponent implements OnInit {
 
     // Patterns
     // length of < 3 is also accepted in order to provide more accurate validation error messages
-    readonly shortNamePattern = RegExp('(^(?![\\s\\S]))|^[a-zA-Z][a-zA-Z0-9]*$|' + SHORT_NAME_PATTERN); // must start with a letter and cannot contain special characters
-    readonly titleNamePattern = RegExp('^[a-zA-Z0-9-_ ]+'); // must only contain alphanumeric characters, or whitespaces, or '_' or '-'
+    readonly SHORT_NAME_REGEX = RegExp('(^(?![\\s\\S]))|^[a-zA-Z][a-zA-Z0-9]*$|' + SHORT_NAME_PATTERN); // must start with a letter and cannot contain special characters
+    readonly TITLE_NAME_REGEX = RegExp(TITLE_NAME_PATTERN);
```

Comment:
There is a console error with the current RegEx, while trying to fix it I have noticed that this RegEx was duplicated, so I introduced a constant for it.
I reverted the change to the RegEx since some tests failed, the adjustment I had in mind was not semantically equal. 

I left the changes since now the RegEx at least only needs to be fixed in one place.